pages/common/code.md

```diff
@@ -0,0 +1,16 @@
+# code
+
+> Visual Studio Code   
+> Code editing. Redefined.
```

Comment:
This line seems to be more subjective and less informative. Is it absolutely needed ?

Reply:
Probably not. I picked it up from their website. I'll remove that.